test/test.js

```diff
@@ -142,4 +148,56 @@ test('The server', [
     server.kill();
     server.on('exit', function() { done(); });
   }],
+])})
```

Comment:
The test runner was changed to Mocha. When you rebase this, could you move these tests into their own `.spec.js` file?